Shopify/dawn
# [Image banner] Add ambient movement to background

### PR Summary: 

Adds an option for ambient animation to the Image Banner section's background image. 

https://user-images.githubusercontent.com/1202812/221939239-9930e1d4-f520-420b-93c2-4f5baf8001ca.mov

### Why are these changes introduced?

Closes #2339. 

### What approach did you take?

This implements a new dropdown menu [as shown in the ticket](#2339). When "Ambient movement" is selected, a CSS class is applied to the media elements within the Image Banner, triggering a long-duration CSS animation that slowly moves the image around within its frame. The CSS for this is defined in `base.css` so that we can theoretically reuse it for the Image with Text (#2340) and Slideshow (#2341) sections. 

As this feature is purely aesthetic, it supports the visitor's `prefers-reduced-motion` preference.

### Visual impact on existing themes

None, this is an optional setting that only adds new functionality. 

### Testing steps/scenarios

1. [Visit the demo store](https://os2-demo.myshopify.com/admin/themes/139450515478/editor). 
2. Add and select an Image Banner. 
3. Look for the new "Image behavior" dropdown underneath the new "Animations" header in the sidebar. 
4. Select "Ambient movement" from that dropdown. 
5. Add various images, verify that they appear as expected.
6. Try various breakpoints, verify that they appear as expected.

### Checklist
